octobot_trading/exchanges/parser/orders_parser_ccxt.py

```diff
@@ -0,0 +1,66 @@
+from octobot_trading.enums import (
+    ExchangeOrderCCXTColumns as CCXTOrderCols,
+    ExchangeConstantsOrderColumns as OrderCols,
+    OrderStatus,
+    TraderOrderType,
+    TradeOrderType,
+)
+import octobot_trading.exchanges.parser.orders_parser as orders_parser
+
+
+class CCXTOrdersParser(orders_parser.OrdersParser):
+    """
+    override CCXTOrdersParser if you add support for a CCXT exchange
+    dont use GenericCCXTOrdersParser as a base
+
+    parser usage:   parser = CCXTOrdersParser(exchange)
+                    orders = await parser.parse_orders(raw_orders)
+                    order = await parser.parse_order(raw_order)
+    """
+
+    TIMESTAMP_KEYS: list = [CCXTOrderCols.TIMESTAMP.value]
+    STATUS_KEYS: list = [CCXTOrderCols.STATUS.value]
+    ID_KEYS: list = [CCXTOrderCols.ID.value]
+    SYMBOL_KEYS: list = [CCXTOrderCols.SYMBOL.value]
+    SIDE_KEYS: list = [CCXTOrderCols.SIDE.value]
+    TYPE_KEYS: list = [CCXTOrderCols.TYPE.value]
+    TAKER_OR_MAKER_KEYS: list = [CCXTOrderCols.TAKER_OR_MAKER.value]
+    PRICE_KEYS: list = [
+        CCXTOrderCols.AVERAGE.value,  # first try average as its more accurate
+        CCXTOrderCols.PRICE.value,
+        CCXTOrderCols.STOP_PRICE.value,  # use only if others are missing
+    ]
```

Comment:
I don't think it's a good approach, I think we should only use the standard one (the only one documented in https://docs.ccxt.com/en/latest/). We shouldn't brute force util we find a value. It may be a totally different value with some exchanges.

Reply:
oh I thought stop price is default, removed it.
No its optional for the generic parser